packages/edit-site/src/components/site-hub/index.js

```diff
@@ -41,32 +43,33 @@ const SiteHub = forwardRef( ( props, ref ) => {
 	const disableMotion = useReducedMotion();
 	const { setCanvasMode } = unlock( useDispatch( editSiteStore ) );
 	const { clearSelectedBlock } = useDispatch( blockEditorStore );
-	const isBackToDashboardButton = canvasMode === 'view';
-	const showLabels = canvasMode !== 'edit';
-	const siteIconButtonProps = isBackToDashboardButton
-		? {
-				href: dashboardLink || 'index.php',
-				'aria-label': __( 'Go back to the dashboard' ),
-		  }
-		: {
-				label: __( 'Open Navigation Sidebar' ),
-				onClick: () => {
-					clearSelectedBlock();
-					setCanvasMode( 'view' );
-				},
-		  };
+	const siteIconButtonProps = {
+		label: __( 'Open Navigation Sidebar' ),
+		onMouseDown: () => {
+			if ( canvasMode === 'edit' ) {
+				clearSelectedBlock();
+				setCanvasMode( 'view' );
+			}
+		},
+	};
 	const siteTitle = useSelect(
 		( select ) =>
 			select( coreStore ).getEntityRecord( 'root', 'site' )?.title,
 		[]
 	);
 
+	const controls = useAnimation();
+	useEffect( () => {
+		controls.set( { layout: true } );
+	} );
```

Comment:
You want to do this, only on mount right? I guess you're missing `[]` as a second argument of useEffect

Reply:
yeah, the linter was complaining